.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: python
 sudo: false
 python:
-- 3.5
```

Comment:
This Travis config doesn't actually run any more.  I intend to replace this with GitHub actions workflows.  I guess continuing to support python 3.5 is a goal, but supporting typing in 3.5 should not be a goal IMO.